corehq/apps/app_manager/static/app_manager/js/visit-scheduler.js

```diff
@@ -39,7 +133,35 @@ var VisitScheduler = (function () {
             self.saveButton.fire('change');
         };
 
-        self.formSchedule = FormSchedule.wrap(params.schedule, self);
+        self.validate = function(){
```

Comment:
Yeah, couldn't find any good examples, so I made it up. Might be useful to extract this into `saveButton` if this will become a common pattern.